package.json

```diff
@@ -13,7 +13,7 @@
     "formatDeclarations": "prettier --ignore-path *.js --write dist/*.d.ts",
     "prepublishOnly": "yarn test && yarn build && yarn formatDeclarations",
     "test": "node scripts/update-test-tsconfig.js && prettier -c {bench,lib,test}/**/*.ts && tsc --project tsconfig.test.json --outDir ./testDist && rimraf ./testDist",
-    "test:fix": "prettier --write {bench,lib,test}/**/*.ts && tsc --project tsconfig.test.json --outDir ./testDist && rimraf ./testDist",
+    "test:fix": "prettier --write {bench,lib,test}/**/*.ts",
```

Comment:
Because `test:fix` script is expected to do more than `test` script, it makes sense to still run `tsc` for type-checking. Therefore I suggest returning your change and adding `node scripts/update-test-tsconfig.js &&` in front of `test:fix` script.

WDYT?

Reply:
@Beraliv You mean the only difference b/w `test` and `test:fix` should be that the former checks prettier formatting and the latter fixes prettier formatting? If so, what's the benefit of type checking within `test:fix`, is it so that contributors can just run a single script that does everything?

Reply:
Cool, I've added `node scripts/update-test-tsconfig.js` to the script in [this commit](https://github.com/ts-essentials/ts-essentials/pull/417/commits/90ee600486bc76d4e0683f8b5bf33f0f1ad5f4e8).

Reply:
@Beraliv I think we should also update the default `tsconfig.test.json` because now every time we run `test:fix` it would show a change in that file. WDYT?

<img width="1487" alt="image" src="https://github.com/user-attachments/assets/6e39f72e-1a31-48c2-af14-383071ef64f2">

Reply:
Yeah, I already saw it, added it to the main branch in https://github.com/ts-essentials/ts-essentials/commit/bc00a4f773967b4d23a37cee753c9cce3ef72b90